frontend/src/modules/admin/modules/users/pages/users.page.vue

```diff
@@ -117,17 +162,32 @@ const loadMore = () => {
   fetchUsers();
 };
 
-const roleDisplay = (roles: string[]) => {
-  const role = roles?.[0];
+const projects = computed(() => treeMap(segmentStore.projectGroups.list));
```

Comment:
We have a function in segment.js to get segment name by id so you dont need to do this stuff.